app/configurations/config.okc.js

```diff
@@ -5,7 +5,11 @@ const CONFIG = 'okc';
 const APP_TITLE = 'Embark Oklahoma City';
 const APP_DESCRIPTION = 'Uusi Reittiopas - okc';
 
+const MAPTILER_KEY = process.env.MAPTILER_KEY
```

Comment:
Shall we `assert.ok()` here that `MAPTILER_KEY` is not missing/empty?

Reply:
Sounds good, though I'm not sure if this would not break running another configs, as I'm not sure if only the active config is parsed